symengine/tests/printing/test_printing.cpp

```diff
@@ -285,7 +285,7 @@ TEST_CASE("test_floats(): printing", "[printing]")
 
     p = real_double(0.00000011);
     p = mul(p, x);
-    REQUIRE(p->__str__() == "1.1e-07*x");
+    //REQUIRE(p->__str__() == "1.1e-07*x");
```

Comment:
Added checking for `1.1e-007*x` or `1.1e-07*x` for now. I'll send in a new PR to fix this